client/orderbook/orderbook.go

```diff
@@ -673,7 +673,7 @@ func (ob *OrderBook) AddRecentMatches(matches [][2]int64, ts uint64) []*MatchSum
 
 	ob.matchSummaryMtx.Lock()
 	defer ob.matchSummaryMtx.Unlock()
-	ob.matchesSummary = append(newMatches, ob.matchesSummary...)
+	ob.matchesSummary = append(newMatches, ob.matchesSummary...) // nolint:makezero
```

Comment:
False positive, but it could have been allocated like `newMatches := make([]*MatchSummary, len(matches), len(matches)+len(ob.matchesSummary))`, but that requires moving the lock and it's not a helpful optimization.